dev/com.ibm.ws.beanvalidation.v20_fat/test-applications/bvalValueExtractorApp/resources/WEB-INF/beans.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://java.sun.com/xml/ns/javaee"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/beans_1_0.xsd">
+       
+   <!-- TODO this file shouldn't need to be here when CDI 1.1 removes the restriction to need
+             beans.xml to enable an app for CDI -->
```

Comment:
Can we remove this file now, given that the test causes is enabling the cdi-2.0 feature?